themes/doom-tokyo-night-theme.el

```diff
@@ -280,6 +288,8 @@
    (web-mode-css-pseudo-class-face :foreground orange)
    (web-mode-css-property-name-face :foreground blue)
    (web-mode-css-selector-face :foreground teal)
+   (web-mode-css-selector-class-face :foreground keywords :slant 'nil)
```

Comment:
There was an error with the `slant` value being invalid and has been [fixed in #835](https://github.com/doomemacs/themes/pull/835/commits/7635004da1e6e40a0182a6f58a2f97d5e6a7f046).

Just change the value to `normal`, which is valid:

```suggestion
   (web-mode-css-selector-class-face :foreground keywords :slant 'normal)
```